docs/advanced-analytics/postgis.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Geospatial Analytics
+---
+
+# Geospatial Analytics
+
+[PostGIS](https://postgis.net/) extends the capabilities of the PostgreSQL by adding support for storing, indexing, and querying geospatial data. Cloudberry Database supports PostGIS for geospatial analytics.
+
+This document introduces how to compile and build PostGIS for your Cloudberry Database cluster.
+
```

Comment:
Shall we list our PostGIS for the Cloudberry Database GitHub repo here? It is dedicated to CloudberryDB.